arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -327,7 +327,7 @@
             "schema": {
               "$ref": "#/definitions/KeyCredentialsUpdateParameters"
             },
-            "description": "Parameters to update keyCredentials of an existing application."
+            "description": "Parameters to update the keyCredentials of an existing application."
```

Comment:
I'm guess keyCredentials are the Azure AD client key, the passwordCredential is the client password, etc? We should probably clarify that.